pkgs/tools/networking/pwru/default.nix

```diff
@@ -0,0 +1,110 @@
+{ lib
+, stdenv
+, buildGoModule
+, fetchFromGitHub
+, pkg-config
+, llvmPackages_13
+, flex
+, bison
+, buildPackages
+, testers
+, pwru
+}:
+
+let
+  inherit (stdenv.targetPlatform) system;
+  goArchs = {
+    "x86_64-linux" = "amd64";
+    "aarch64-linux" = "arm64";
+  };
+  goArch = goArchs.${system} or (throw "Unsupported platform ${system}");
+
+  pname = "pwru";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "cilium";
+    repo = "pwru";
+    rev = "v${version}";
+    hash = "sha256-9UMuHDPDL8nPdL6U+BHHSXvd0iDUigVOa5Tel7n5L4Q=";
+  };
+
+  libpcap = stdenv.mkDerivation {
```

Comment:
Why do we use their vendored libpcap? We can't just use ours?

Reply:
Because those flags are applied:
```
    cmakeFlags = [
      "-DBUILD_SHARED_LIBS=OFF"
      "-DDISABLE_LINUX_USBMON=ON"
      "-DDISABLE_BLUETOOTH=ON"
      "-DDISABLE_NETMAP=ON"
      "-DDISABLE_DBUS=ON"
      "-DBUILD_WITH_LIBNL=OFF"
    ];
```

Reply:
It shouldn't be a problem if we supply a lib with more features, right?

Reply:
Because of the static build it seems that if libpcap is not compiled that way, when the build of  Go modules occurs in static mode it fails to find libnl.